src/case-monitoring/sub-process.ts

```diff
@@ -67,17 +67,33 @@ class SubProcessTippySupport extends AbstractTippySupport {
     // Target instance.popper. Keep using document for now as the previous popper (when going back to the subprocess after a first venue)
     // may still exist in the DOM of the subprocess view
     const rows = document.querySelectorAll(`#${instance.popper.id} #popover-resources-available > tbody > tr`);
-    for (const [, row] of rows.entries()) {
+    for (const [index, row] of rows.entries()) {
+      const assignBtn = document.querySelector(`#${instance.popper.id} #assign${index}`);
```

Comment:
**suggestion:** I suggest to add `#popover-resources-available` to the selector to avoid side effects.
We may use a more specific selector as we know the button are within the tr `#${instance.popper.id} #popover-resources-available > tbody > tr #<btn-id>`

Reply:
fixed in d58e0cd125330f3dd4245ba564293f06a5d3d8cb